docs/nx/next/init-deployment.md

```diff
@@ -87,38 +90,40 @@ As a rule of thumb, each task here references a target execution via Nx defined
 
 ```yaml
 helm:
-    description: Lint Helm Charts
-    command:
-      - npx nx affected --base="$BASE_SHA" --target=helm-lint
+  description: Lint Helm Charts
+  command:
+    - npx nx affected --base="$BASE_SHA" --target=lint
 ```
 
 `apps/myapp/project.json`
 
 ```yaml
-"helm-lint":
+"lint":
   {
     "executor": "nx:run-commands",
     "options":
       {
-        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
+        "commands": [{ "command": "lint", "forwardAllArgs": false }],
```

Comment:
Helm is still in this command
```suggestion
        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
```

Reply:
yeah this section is outdated. In the actual changes diff, I can see it is correct 😂 
<img width="689" alt="image" src="https://user-images.githubusercontent.com/101654886/230032049-98f3eafe-0dd5-46a0-96f7-8e6a0cde0640.png">